Makefile

```diff
@@ -82,13 +72,23 @@ WEBPACK_DEST_DIRS := public/js public/css
 BINDATA_DEST := modules/public/bindata.go modules/options/bindata.go modules/templates/bindata.go
 BINDATA_HASH := $(addsuffix .hash,$(BINDATA_DEST))
 
+TAGS ?=
+TAGS_SPLIT := $(subst $(COMMA), ,$(TAGS))
+TAGS_EVIDENCE := $(MAKE_EVIDENCE_DIR)/tags
+
+GO_SOURCES := $(wildcard *.go)
+GO_SOURCES += $(shell find cmd integrations models modules routers scripts services vendor -type f -name "*.go" -not -path modules/options/bindata.go -not -path modules/public/bindata.go -not -path modules/templates/bindata.go)
```

Comment:
I see, if TAGS change we need to get rid of bindata.go. 👍

Reply:
Yes, it's done this way so it works regardless whether bindata files already exists or not.